system-components/src/emotion.js

```diff
@@ -2,7 +2,8 @@ import styled from 'react-emotion'
 import System from './System'
 
 const create = new System({
-  createComponent: type => (...args) => styled(type)(...args)
+  createComponent: type => (...args) =>
+    styled(type, { shouldForwardProp: name => name === "innerRef" })(...args)
```

Comment:
It should be this.
```jsx
name => name !== 'ref' && name !== 'theme'
```

Reply:
Nice! I tried that out in a code sandbox and it worked 🎉 https://codesandbox.io/s/4qvpyo8r09

@jxnblk Looks like we didn't need to change `clean-tag` after all :)